wp-content/themes/thinkery/style.css

```
@@ -1485,23 +1486,68 @@ p.entry-meta {
 	margin-bottom: 0;
 }
 
+.footer-widgets div ul li a {
+	color: white;
+	font-family: museo-sans, sans-serif;
+	font-size: .85em;
+	font-weight: 400;
+	text-decoration: none;
+}
+.footer-widgets #fa-icons {
+display: flex;
+justify-content: space-between;
+font-size: 1.5em;
+
+}
+
+.footer-widgets #fa-icons i:focus,
+.footer-widgets #fa-icons i:hover {
+	color: #1b2432;
+}
+
+.footer-widgets a:focus,
+.footer-widgets a:hover {
+	color: #1b2432;
+}
+
+.footer-widgets h3 {
```

```suggestion
.footer-widgets .widget-title {
    text-transform: uppercase;
```
Lets target the class, not the element here (in case we need to change the element down the line for SEO Reasons. Lets also style the capital letter with CSS instead of using uppercase letters in the editor for the better accessibility.

---

Great call, changes applied.